server/database.js

```diff
@@ -0,0 +1,65 @@
+const {getDomain} = require('./domain.js');
+
+//Endpoint to handle requests to access participant data
+//queries databse for a specific url corresponding to a certain group 
+//of users and returns all rows in the evidence table with that url
+//Returns 200 unless there was an error with the database query
+function dataEndpoint(pool, request, response) {
+  const token = request.headers['x-teachermoments-token'];
+  const domain = getDomain(request);
+  const location = `${domain}/teachermoments/turner?KevinTesting20180319`;
+
+  checkAccess(pool, location, token)
+    .then(isTokenAuthorized => {
+      if (isTokenAuthorized) {
+        getData(pool, location)
+          .then(results => {
+            response.set('Content-Type', 'application/json');
+            response.json({
+              evidence: {rows: results} 
+            });
+            return response.status(200).end();
+          });
+      }
+      else{
+        console.log('Researcher is not authorized to see this data');
+        return response.status(405).end();
```

Comment:
`405` is a bit different (https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405) `403` might be more what you want.

Reply:
hmmm. Good catch. I actually haven't thought too deeply on the status codes. Fixed!